README.md

```diff
@@ -11,9 +11,11 @@ MCprep is an addon dedicated to speeding up the workflow of Minecraft animators
 ## Installing MCprep
 ### Click the link below and download the .zip file (re-zip if auto-unzipped into a folder necessary), install into blender (2.80 through 3.5 supported)
 
+**Note: MCprep is developed with the assumption that our users have a valid, legal copy of Minecraft. As such, Moo-Ack! Productions reserves the right to deny support if one is found to use a pirated copy of Minecraft**
```

Comment:
If the purpose is to be more legally airtight, then we should rephrase "we assume" to "you must" as I do below. Otherwise, it almost further emphasizes the license as optional by saying the downside is just no user-support.

```suggestion
**Note: To use MCprep, you must have a valid, legal copy of Minecraft to comply with Mojang's terms of service. No support is provided for anyone using MCprep without a valid Minecraft license**
```